### ISSUE TITLE
Device Text Zoom Functionality

### ISSUE THREAD

**Issue by ehorodyski-ionic** [ionic-team/capacitor-plugins#3]
### Plugin

ScreenReader (or a new plugin?)

### Description

While working with a member of the Capacitor community, there was a strong desire for the ability to have Capacitor expose functionality related to device zoom size.

Exposing this functionality would enable this member to implement accessibility best practices into their design for users with visual impairments.

### Platform(s)

- iOS
- Android

### Preferred Solution

It would be ideal to have a Capacitor plugin expose the following functionality:

- `getTextZoom(): Promise<number>`
  Return the device's current text zoom percent value for the WebView.
- `setTextZoom(number): Promise<void>` 
  Set the current text zoom percent value for the WebView.
- `updateTextZoom(): Promise<number>`
  Retrieve the device's preferred text zoom from settings and apply it to the WebView.
- `usePreferredTextZoom(boolean): Promise<void>`
  If `false`, set the WebView zoom percentage to 100. If `true`, retrieve the device's preferred text zoom from settings and apply it to the WebView
  

### Alternatives

iOS WebKit provides a mechanism to obtain the system font in CSS, but it appears this is not the case with Android.

`phonegap-mobile-accessibility` is a Cordova plugin [that provides this functionality](https://github.com/phonegap/phonegap-mobile-accessibility) last updated 4 years ago. The lack of maintenance is a concern of the community member.
